ci/utils/fetch_polars_versions.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+import argparse
+import json
+import ssl
+import urllib.request
+
+import certifi
+import yaml
+from packaging.specifiers import SpecifierSet
+from packaging.version import Version
+
+
+def get_polars_specifier(deps_yaml_path):
+    with open(deps_yaml_path, "r") as f:
+        deps = yaml.safe_load(f)
+
+    try:
+        includes = deps["files"]["all"]["includes"]
+        if "run_cudf_polars" not in includes:
+            raise KeyError
```

Comment:
Raise an instance of the class, not the class itself.
```suggestion
            raise KeyError()
```